.github/workflows/submodulesync.yaml

```diff
@@ -17,10 +17,11 @@ jobs:
       - name: Update sub modules
         run: |
           git submodule update --remote
-          (cd src/lima && git fetch --tags)
-          TAG=`cd src/lima && git describe --tags $(git rev-list --tags --max-count=1)`
-          echo "Pulling changes from release: $TAG"
-          (cd src/lima && git checkout $TAG)
+#          TODO: Track back release once lima has v0.17.3
```

Comment:
nit: next Lima release looks to be v0.18.0: https://github.com/lima-vm/lima/milestones